packages/contracts/src/SpawnTileProgram.sol

```diff
@@ -5,52 +5,48 @@ import { WorldConsumer } from "@latticexyz/world-consumer/src/experimental/World
 import { System } from "@latticexyz/world/src/System.sol";
 import { WorldContextConsumer } from "@latticexyz/world/src/WorldContext.sol";
 
-import { MAX_PLAYER_ENERGY } from "@dust/world/src/Constants.sol";
-import { ISpawn, SpawnContext } from "@dust/world/src/ProgramHooks.sol";
+import { HookContext, IAttachProgram, IDetachProgram, ISpawn } from "@dust/world/src/ProgramHooks.sol";
 
-import {
-  AttachProgramContext, DetachProgramContext, IAttachProgram, IDetachProgram
-} from "@dust/world/src/ProgramHooks.sol";
 import { Energy } from "@dust/world/src/codegen/tables/Energy.sol";
-import { ObjectPhysics } from "@dust/world/src/codegen/tables/ObjectPhysics.sol";
 import { IWorld } from "@dust/world/src/codegen/world/IWorld.sol";
 import { EntityId } from "@dust/world/src/types/EntityId.sol";
 import { ObjectTypes } from "@dust/world/src/types/ObjectType.sol";
 
 import { Admin } from "./codegen/tables/Admin.sol";
-import { Contribution } from "./codegen/tables/Contribution.sol";
+
 import { ForceFieldDamage } from "./codegen/tables/ForceFieldDamage.sol";
+import { ForceFieldEnergy } from "./codegen/tables/ForceFieldEnergy.sol";
 import { SpawnEnergyConsumed } from "./codegen/tables/SpawnEnergyConsumed.sol";
 import { getForceField } from "./utils/getForceField.sol";
 
 uint128 constant MIN_ENERGY_THRESHOLD_TO_SPAWN = 200_000_000_000_000_000_000;
```

Comment:
is this the threshold we want for the monument?

Reply:
just used the one for perm city

Reply:
maybe just 500k is fine, I think this is 2 million